jekyll/_cci2/build-project.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: classic-docs
+title: "Hello World"
+short-title: "Hello World"
+description: "First project on CircleCI 2.0"
+categories: [getting-started]
+order: 4
+---
+
+This document describes how to configure your project to run on CircleCI 2.0. 
+
+## Prerequisites
+
+- Complete the steps in the [Join Beta & Sign Up]({{ site.baseurl }}/2.0/first-steps/) document.  
+
+## Steps
+
+1. Create a directory called `.circleci` in the root directory of your local GitHub or Bitbucket code repository. 
+
+2. Create a `config.yml` file in the `.circleci` directory with the following lines, replacing *project root directory* with your project directory and *language:version* with your programming language and version number. See the [CircleCI Images doc]({{ site.baseurl }}/2.0/circleci-images/) for a complete list of languages and versions.
+
+```
+version: 2
+jobs:
+  build:
+    working_directory: ~/<project root directory>
+    docker:
+      - image: circleci/<language>:<version TAG>
+      - environment:
```

Comment:
If using this I'd consider outputting it in the steps. Otherwise, consider omitting for Hello World